programmers-coding-test/soongu/greedy/큰 수의 법칙1.js

```diff
@@ -0,0 +1,34 @@
+const rl = require("readline-sync");
+
+let [ n, m, k] = rl.question("N, M, K 입력해주세요.\n>> ").split(',');
+
+
+let data = rl.question("data 입력해주세요.\n>> ").split('');
+
+for (let i = 0; i < data.length; i++) {
+  data[i] = parseInt(data[i]);
+}
+
+n = parseInt(n);
+m = parseInt(m);
+data.sort((a, b) => a - b);
+
+let first = data[n - 1];
+let second = data[n - 2];
```

Comment:
내림차순으로 정렬하고 0번지에 접근하는 것이 가독성에 좋을 듯 하다.  minus 연산도 빠지므로 (사실 거기서 거기지만) 효율도 좋아진다.
또한 first, second는 변하지 않는 값이므로 const가 좋겠다.
```suggestion
data.sort((a, b) => b - a);

const first = data[0];
const second = data[1];
```